packages/live-preview-sdk/src/csm/encode.ts

```diff
@@ -0,0 +1,22 @@
+import { vercelStegaDecode, vercelStegaEncode } from '@vercel/stega';
+
+export type SourceMapMetadata = {
+  origin: string;
+  href: string;
+  cf: {
+    space: string;
+    environment: string;
```

Comment:
can `environment` be undefined?

Reply:
Backend always returns it and we should always set it unless a bug happened somewhere.